metadata.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "puppetlabs-motd",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Commits shouldn't update the metadata.json version since we don't know what will be released next / higher likelyhood of conflicts.